Example/README.md

```diff
@@ -0,0 +1,21 @@
+![Yoshi](../Images/Yoshi_logo.jpg)
```

Comment:
No. Let's reserve the image for the main README. We didn't use the banner for the best practices document in our style guide so let's keep it consistent for now.